Cargo.toml

```diff
@@ -31,6 +31,8 @@ rand = "0.7.2"
 serde = { version = "1.0.104", features = ["derive"] }
 serde_json = "1.0.44"
 toml = "0.5.5"
+escaper = "0.1.0"
```

Comment:
You had added an `escaper` at #11 before, bro, why so many `escaper`? LOL~

Reply:
I'll push a new PR.

Reply:
It doesn't matter, force push to override is okay.

Reply:
It's a mess, new PR here #17